bs-config.js

```diff
@@ -0,0 +1,17 @@
+/*
+ |--------------------------------------------------------------------------
+ | Browser-sync config file
+ |--------------------------------------------------------------------------
+ |
+ | For up-to-date information about the options:
+ |   http://www.browsersync.io/docs/options/
+ |
+ | There are more options than you see here, these are just the ones that are
+ | set internally. See the website for more info.
+ |
+ |
+ */
+module.exports = {
+  files: ['styleguide/_themes/derek/scss/**/*.scss', 'styleguide/_themes/**/**/*.scss'],
```

Comment:
Now that we have babel watch setup, can we add these paths? 
`styleguide/js/*.js`
`styleguide/js/*/*/*.js